paimon-format/src/main/resources/META-INF/NOTICE

```diff
@@ -26,6 +26,7 @@ This project bundles the following dependencies under the Apache Software Licens
 - org.apache.parquet:parquet-format-structures:1.12.3
 - org.apache.parquet:parquet-jackson:1.12.3
 - commons-pool:commons-pool:1.6
+- com.github.luben:zstd-jni:1.15.0-1
```

Comment:
This is not apache license: https://github.com/luben/zstd-jni/blob/master/LICENSE

Reply:
fixed